common/src/main/java/net/neoforged/gradle/common/runs/ide/IdeRunIntegrationManager.java

```diff
@@ -210,6 +214,34 @@ private TaskProvider<?> createIdeBeforeRunTask(Project project, String name, Run
             return ideBeforeRunTask;
         }
         
+        private void addEclipseCopyResourcesTasks(EclipseModel eclipse, Run run) {
+            for (SourceSet sourceSet : run.getModSources().get()) {
+                final Project sourceSetProject = SourceSetUtils.getProject(sourceSet);
+
+                final String taskName = CommonRuntimeUtils.buildTaskName("eclipseCopy", sourceSet.getProcessResourcesTaskName());
+                final TaskProvider<?> eclipseResourcesTask;
+
+                if (sourceSetProject.getTasks().findByName(taskName) != null) {
+                    eclipseResourcesTask = sourceSetProject.getTasks().named(taskName);
+                }
+                else {
+                    eclipseResourcesTask = sourceSetProject.getTasks().register(taskName, Copy.class, task -> {
+                        final TaskProvider<ProcessResources> defaultProcessResources = sourceSetProject.getTasks().named(sourceSet.getProcessResourcesTaskName(), ProcessResources.class);
+                        task.from(defaultProcessResources.get().getDestinationDir());
+                        Path outputDir = eclipse.getClasspath().getDefaultOutputDir().toPath();
+                        if (outputDir.endsWith("default")) {
```

Comment:
Will add comment from where this sources, but not sure why it isn't replaced with correct value automatically (because it was last year when i was first testing this)

Testing: [default value in gradle](https://github.com/gradle/gradle/blob/f796b166368355a4c62fbbfb3157bd29c3f2a25c/platforms/ide/ide/src/main/java/org/gradle/plugins/ide/eclipse/internal/EclipsePluginConstants.java#L21) is `bin/default`
This can be changed using `eclipse.classpath.defaultOutputDir = file('ecbuild')`

Output from default:
```
TASK CREATION defaultOutputDir: P:\data\github\BlockUI\bin\default
TASK CFG copy from: P:\data\github\BlockUI\build\resources\main copy to: P:\data\github\BlockUI\bin\main | defaultOutputDir: P:\data\github\BlockUI\bin\default
TASK CREATION defaultOutputDir: P:\data\github\BlockUI\bin\default
TASK CREATION defaultOutputDir: P:\data\github\BlockUI\bin\default
```

Output from changed:
```
TASK CREATION defaultOutputDir: P:\data\github\BlockUI\ecbuild
TASK CFG copy from: P:\data\github\BlockUI\build\resources\main copy to: P:\data\github\BlockUI\ecbuild\main | defaultOutputDir: P:\data\github\BlockUI\ecbuild 
TASK CREATION defaultOutputDir: P:\data\github\BlockUI\ecbuild
TASK CREATION defaultOutputDir: P:\data\github\BlockUI\ecbuild
```

TASK CREATION is called when task is being registered, TASK CFG is called from within configuration action of said task